docs/book/src/user/quick-start.md

```diff
@@ -172,6 +172,17 @@ and `kubeadm` control-plane providers.
 
 #### Initialization for common providers
 
+<aside class="note">
+
+<h1>ClusterClass</h1>
+
+If you want to deploy a Cluster with ClusterClass, the `ClusterTopology` feature has to be enabled.
+```bash
+export CLUSTER_TOPOLOGY=true
+```
```

Comment:
Let's make this more generic by having a sub paragraph named "enabling feature gated features" that links to the feature gate doc, but keeping ClusterClass as example

Reply:
Done, ptal